content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -327,8 +327,8 @@ Enforce all invariants, now that the service account has been set up:
 ```sh
 # Enforce invariants that will finalize the service account setup
 mi2 instance check -enforce -label service-account
-# Make sure all invariants are applied
-mi2 instance check -enforce
+# Make sure all invariants are applied, including inviting the customer admin again
+mi2 instance check -enforce -customer-admin-email $CUSTOMER_ADMIN_EMAIL
```

Comment:
should add a note the admin email should match the one during creation

Reply:
```suggestion
# Note that $CUSTOMER_ADMIN_EMAIL must match the one the Cloud instance was initially created with
mi2 instance check -enforce -customer-admin-email $CUSTOMER_ADMIN_EMAIL
```